grakn-test/test-integration/src/test/java/ai/grakn/test/graql/reasoner/BenchmarkTests.java

```diff
@@ -0,0 +1,226 @@
+/*
+ * Grakn - A Distributed Semantic Database
+ * Copyright (C) 2016  Grakn Labs Limited
+ *
+ * Grakn is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Grakn is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
+ */
+
+package ai.grakn.test.graql.reasoner;
+
+import ai.grakn.Grakn;
+import ai.grakn.GraknSession;
+import ai.grakn.GraknSystemProperty;
+import ai.grakn.GraknTx;
+import ai.grakn.GraknTxType;
+import ai.grakn.Keyspace;
+import ai.grakn.client.BatchExecutorClient;
+import ai.grakn.client.GraknClient;
+import ai.grakn.concept.ConceptId;
+import ai.grakn.concept.RelationshipType;
+import ai.grakn.concept.Role;
+import ai.grakn.graql.GetQuery;
+import ai.grakn.graql.Graql;
+import ai.grakn.graql.InsertQuery;
+import ai.grakn.graql.Pattern;
+import ai.grakn.graql.Var;
+import ai.grakn.graql.admin.Answer;
+import ai.grakn.test.rule.EngineContext;
+import com.google.common.collect.HashMultimap;
+import com.google.common.collect.Multimap;
+import com.google.common.io.Files;
+import java.io.File;
+import java.nio.charset.Charset;
+import java.util.List;
+import java.util.Random;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static ai.grakn.graql.Graql.var;
+import static ai.grakn.util.SampleKBLoader.randomKeyspace;
+import static org.junit.Assert.assertEquals;
+
+public class BenchmarkTests {
```

Comment:
Since this is using a real engine, we should call it `BenchmarkIT`. Unfortunately, putting it in the `test-integration` module isn't enough to get it labelled an integration test :(